Fabric.Authorization.API/Modules/UsersModule.cs

```diff
@@ -37,31 +38,31 @@ public class UsersModule : FabricModule<User>
             Get("/permissions", async _ => await GetCurrentUserPermissions().ConfigureAwait(false), null,
                 "GetUserPermissions");
 
-            Post("/{userId}/AdditionalPermissions",
+            Post("/{subjectId}/AdditionalPermissions",

```

Comment:
Do we also need identityProvider here? I think these endpoints could be used by actors who are not the subject/target of the permission changes.

Reply:
You are getting ahead of me here. That is what I am going to suggest in the api changes to support identity provider that I have not submitted to the team yet.